### PR TITLE
Add docstrings to spider classes 

### PR DESCRIPTION
**Pull request by aguilerapy**
I will create multiple PRs for #347. For a simpler review, I will try to have each reviewer verify the spiders he wrote.

In this first PR, most of the spiders were written by @yolile and me.

### CODE REVIEW

File: kingfisher_scrapy/spiders/nepal_dhangadhi.py [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
 
 class NepalDhangadhi(SimpleSpider):
+    """
+    Spider arguments
```
Comment (yolile) [inline kingfisher_scrapy/spiders/nepal_dhangadhi.py:12]: maybe we can add this https://ims.susasan.org/dhangadhi/about as "Bulk Download Documentation"

File: kingfisher_scrapy/spiders/nepal_portal.py [hunk 2]
```diff
@@ -7,6 +7,11 @@
 
 
 class NepalPortal(SimpleSpider):
+    """
+    Spider arguments
```
Comment (yolile) [inline kingfisher_scrapy/spiders/nepal_portal.py:11]: maybe we can add http://ppip.gov.np/downloads as "Bulk Download Documentation"

File: kingfisher_scrapy/spiders/portugal.py [hunk 3]
```diff
@@ -8,6 +8,13 @@
 
 
 class Portugal(ZipSpider):
+    """
+    API documentation
+        https://dados.gov.pt/pt/apidoc/
```
Comment (yolile) [inline kingfisher_scrapy/spiders/portugal.py:13]: this page also have a swagger documentation at the bottom

File: kingfisher_scrapy/spiders/uruguay_historical.py [hunk 4]
```diff
@@ -4,6 +4,13 @@
 
 
 class UruguayHistorical(ZipSpider):
+    """
+    API documentation
```
Comment (yolile) [inline kingfisher_scrapy/spiders/uruguay_historical.py:8]: this is not an API so maybe we should call it "Bulk Download Documentation"

**Comment by jpmckinney** [comment]
Woohoo, https://kingfisher-collect.readthedocs.io/en/latest/spiders.html is starting to look good.

**Pull request merged**